### ISSUE TITLE
o:validateMultiple re-triggering PostConstruct

### ISSUE THREAD

**Issue by sharpedavid** [omnifaces/omnifaces#120]
I am using Omnifaces 1.10, JSF 2.1 (Mojarra 2.1.6), and PrimeFaces 3.5.28.

I am using `o:validateMultiple`, and it is working, but it is re-triggering my bean's `@PostConstruct` method on submit. I do not think that it should be re-triggered.

When I replace the same validator with `o:validateAllOrNone` to compare the behaviour, it still works, but the bean's `@PostConstruct` is not re-triggered. I think this suggests that my configuration is mostly correct.

Here is a code snippet:

```xml
 <!--<o:validateAllOrNone 
    id="allOrNone" components="areaCode phoneNumber effectiveFromDate_phoneNum" />-->

<o:validateMultiple
    id="allOrNone" components="areaCode phoneNumber effectiveFromDate_phoneNum" 
    validator="#{addProviderManager.validateMandatoryPhone}" />

<p:commandButton value="Submit" />
```

More details:
- The backing bean is a `@ManagedBean` and `@ViewScoped`.
- areaCode and phoneNumber  are `p:inputText` and effectiveFromDate_phoneNum is a `p:calendar`. 
- The components are nested in a form, and no ajax behaviour is enabled on any of them. 
- `validateMandatoryPhone` is not doing anything: it's stubbed to just return `false`.

Again, to emphasize, everything seems to be working, but `o:validateMultiple` is re-triggering the `@PostConstruct` method, while `o:validateAllOrNone` is not. I do think the `@PostConstruct` method should be re-triggered.

I can provide additional details, files, etc. Please just ask.

**Comment by sharpedavid** [comment]
Tomorrow I'll try moving the validator method out of the backing bean into its own class. I'll report back.

**Comment by BalusC** [comment]
This is a Mojarra bug which is fixed in 2.1.18. Upgrade Mojarra.

Related issues/topics:
- https://java.net/jira/browse/JAVASERVERFACES-1665
- https://java.net/jira/browse/JAVASERVERFACES_SPEC_PUBLIC-787
- http://stackoverflow.com/questions/3342984/jstl-in-jsf2-facelets-makes-sense
